packages/core/src/utilities/ui.ts

```diff
@@ -100,10 +101,14 @@ export const relativePosition = (
     } else {
       top = targetHeight
       left = target.offsetWidth - elementDimensions.width
+
+      if (leftMenu) {
```

Comment:
why was this needed?

Reply:
It was aligning only for the right side, not working as expected

Reply:
hm, alright let's keep this change in mind, in the future, this methods are really hard to tweak.